client/my-sites/theme/main.jsx

```diff
@@ -563,7 +312,7 @@ const ThemeSheetWithOptions = ( props ) => {
 	return (
 		<ConnectedThemeSheet { ...props }
 			siteId={ siteId }
-			theme={ props /* TODO: Have connectOptions() only use theme ID */ }
+			theme={ props.theme /* TODO: Have connectOptions() only use theme ID */ }
```

Comment:
Let's destructure along wit the other `props`, ie on [line 297](https://github.com/Automattic/wp-calypso/pull/10748/files#diff-f576b3c13daa17f9ccee514c4c46bc07R297).